rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/moderationrequest/Sw360ModerationRequestService.java

```diff
@@ -447,4 +455,34 @@ public Integer getOpenCriticalCrCountByGroup(String group) {
             return 0;
         }
     }
+
+    public RequestStatus deleteModerationRequestInfo(@NotNull User sw360User, @NotNull String id,
+			@NotNull ModerationRequest moderationRequest, ModerationState moderationState)
```

Comment:
Parameter `moderationState` should be calculated from `moderationRequest`. Please remove.